content/apidocs-mxsdk/apidocs/team-server-api.md

```diff
@@ -4,203 +4,411 @@ category: "API Documentation"
 Accept: */*
-Mendix-Username: richard.ford51@example.com
-Mendix-ApiKey: 26587896-1cef-4483-accf-ad304e2673d6
+Authorization: MxToken hZUPhAV4ELPrRm7U7JAKf5BnxJk6q7dcsvFdw6ZR4wRYdv7egHjwHEYBwXY4RkSZrAWde3XqVAQkxZNPysvHcpquA9sK9bsKmcTN
 ```
 
-#### 3.1.2 Output
+#### 4.1.2 Response Headers
 
-List of objects with the following key-value pairs:
+|Name|Value|
+|---|---|
+|Content-Type|`application/json; charset=utf-8`|
 
-*   _Name_ (String) : Name of the branch. This is 'trunk' for the main line or a specific branch name.
-*   _DisplayName_ (String) : Visible name in the Developer Portal. For the trunk, this is 'Main line'.
-*   _LatestRevisionNumber_ (Long) : Number of the latest revision.
-*   _LatestRevisionMendixVersion_ (String) : Version string of the Mendix version of the app in this revision.
+#### 4.1.3 Response Payload
 
-##### 3.1.2.1 Error Codes
+List of objects with the following key-value pairs:
 
-| HTTP Status | Error code | Description |
-| --- | --- | --- |
-| 400 | INVALID_APPID | Invalid AppId. |
-| 404 | APP_NOT_FOUND | App not found. |
+|Name|Type|Description|
+|---|---|---|
+|`projectId`|String|The project ID (also known as app ID) of the Mendix project.|
+|`type`|String|The type of repository. At the moment this will be either `"svn"` or `"git"`, but later on other repository types may be introduced.|
+|`url`|String|The URL of the repository.|
+|`isMendixHosted`|Boolean|Whether the repository is Mendix-hosted (i.e. a Team Server repository) or is using a custom repository URL. Note: If a repository is not Mendix-hosted (i.e. `isMendixHosted` is `false`), the other operations of the Team Server API will not work for this repository.|
+|`defaultBranchName`|String (optional)|The default branch name of the repository. Only available for Mendix-hosted projects. The value will be `"trunk"` for Subversion repositories and `"main"` for Git repositories.|
 
-##### 3.1.2.2 Example
+##### 4.1.3.1 Payload Example
 
 ```json
-[{
-     "Name" :  "trunk" ,
-     "LatestRevisionNumber" :  9 ,
-     "LatestRevisionMendixVersion" :  "5.6.0" ,
-     "DisplayName" :  "Main line"
-},{
-     "Name" :  "statistical functions" ,
-     "LatestRevisionNumber" :  13 ,
-     "LatestRevisionMendixVersion" :  "5.6.0" ,
-     "DisplayName" :  "statistical functions"
-}]
+{
+  "projectId": "c0af1725-edae-4345-aea7-2f94f7760e33",
+  "type": "svn",
+  "url": "https://teamserver.sprintr.com/c0af1725-edae-4345-aea7-2f94f7760e33/",
+  "isMendixHosted": true,
+  "defaultBranchName": "trunk"
+}
 ```
 
-### 3.2 Retrieve Branch {#retrieve-branch}
+##### 4.1.3.2 Error Codes
+
+| HTTP Status | Title | Detail |
+| --- | --- | --- |
+| 400 | Bad Request | Invalid app ID |
+| 401 | Unauthorized | Invalid token |
+| 403 | Forbidden | Access denied |
+| 404 | Not Found | Repository not found |
+| 500 | Internal Server Error | Something went wrong |
+
+Error Response format and examples are given in section: [Error response format](#3-error-response-format)
```

Comment:
Need to add anchors {#XXX} manually to the linked sections, and use the added anchors here for cross-references.

Reply:
Done